occurrences/tests/test_notifications.py

```diff
@@ -208,11 +208,21 @@ def test_cancel_occurrence_notification(
     notification_template_cancel_occurrence_en,
     notification_template_cancel_occurrence_fi,
 ):
-    study_groups = StudyGroupFactory.create_batch(3)
-    for s in study_groups:
-        EnrolmentFactory(study_group=s, occurrence=occurrence, person=s.person)
+
+    for status in Enrolment.STATUSES:
+        for s in StudyGroupFactory.create_batch(2):
+            EnrolmentFactory(
+                study_group=s, occurrence=occurrence, person=s.person, status=status
+            )
```

Comment:
Create 2 enrolments with each status code.  Add the enrolments to occurrence. Enrolment needs an unique study group.